doc/src/manual/parallel-computing.md

```diff
@@ -184,10 +184,10 @@ julia> function make_jobs(n)
 
 julia> n = 12;
 
-julia> @schedule make_jobs(n); # feed the jobs channel with "n" jobs
+julia> @async make_jobs(n); # feed the jobs channel with "n" jobs
 
 julia> for i in 1:4 # start 4 tasks to process requests in parallel
-           @schedule do_work()
+           @async do_work()
```

Comment:
`round(exec_time,2))` a few lines below doesn't work on 1.0 should be `round(exec_time; digits=2)`

Reply:
Also, the `n = n - 1` needs a `global` in front of it.

Reply:
I have pushed a new commit with those changes. Thanks for the comments, nice catch!